pulsar-common/src/test/java/org/apache/pulsar/common/util/jetty/JettySslContextFactoryWithKeyStoreTest.java

```diff
@@ -16,9 +16,21 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pulsar.common.util.keystoretls;
+
+package org.apache.pulsar.common.util.jetty;
```

Comment:
Please move this to `pulsar-broker-common` module.
Switch to use package name `org.apache.pulsar.jetty.tls`.

Reply:
Fixed.